assignments/models.py

```diff
@@ -121,3 +121,30 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class Student(models.Model):
+    assignment = models.ForeignKey(Assignment, related_name='students')
```

Comment:
I think I added assignment on Student also, so I can get all students related to this assignment easier, and for the purpose of reports, but we can remove it.